### PR TITLE
feat(core): button

### PR DESCRIPTION
**Pull request by kevanantha**
image

### CODE REVIEW

**Comment by faultable** [comment]
mantap bosq

**Comment by kevanantha** [comment]
beda node version pak

**Comment by ri7nz** [comment]
It's awesome

File: package.json [hunk 1]
```diff
@@ -4,12 +4,20 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "clean": "lerna clean --yes",
-    "build:docs": "yarn workspace cra run build:storybook",
+    "compile:core": "yarn workspace @evilfactory/anggun-core compile",
```
Comment (ri7nz) [inline package.json]:
```suggestion
    "compile:core": "lerna run compile --scope=@evilfactory/anggun-core",
```

File: package.json [hunk 2]
```diff
@@ -4,12 +4,20 @@
...
+    "compile:core:watch": "yarn workspace @evilfactory/anggun-core compile:watch",
```
Comment (ri7nz) [inline package.json]:
```suggestion
    "compile:core:watch": "lerna run compile:watch --scope=@evilfactory/anggun-core",
```

File: package.json [hunk 3]
```diff
@@ -4,12 +4,20 @@
...
+    "build:docs": "yarn workspace cra run build",
```
Comment (ri7nz) [inline package.json]:
```suggestion
    "build:docs": "lerna run build",
```
when use this is suggestion `without scope`, this is mean you run `build` for all packages.

**Comment by ri7nz** [comment]
**ugvgugofikfknxlfbxcvfhrgxhdgfbvg632e5rex45eo**

File: package.json (hunk 3 above)
Comment (kevanantha) [reply package.json]: btw, if I use this command, I can't see the log

I prefer use the previous one @ri7nz

Comment (ri7nz) [reply package.json]: @kevanantha add options ```--stream``` or ```--paralel```

Comment (kevanantha) [reply package.json]: noted, thanks

**Pull request merged**